atlp-rwanda/atlp-devpulse-bn
# adding cycle while creating and updating trainee

### What does this PR do?

Add cycle while creating and updating trainee

### How should this be manually tested?

1. clone the repo if not yet done by running `git clone https://github.com/atlp-rwanda/atlp-devpulse-bn`
2. run `cd atlp-devpulse-bn`

3. run `git checkout ft-add-cycle-creating-trainee`

4. run ` npm install`

5. run `cp .env.example .env
`

6.run `npm run dev`

// open the link printed at the end of the command
What to expect?
Use **createNewTraineeApplicant** and **updateTraineeApplicant** in apollo server to test it by adding cycle while creating and updating trainee
